### PR TITLE
Fix typos on cmd/kubeadm

### PR DESCRIPTION
**Pull request by yagonobre**
**What this PR does / why we need it**:
https://github.com/kubernetes/kubeadm/issues/1191

```release-note
NONE
```

/kind cleanup

### CODE REVIEW

File: cmd/kubeadm/app/cmd/init.go [hunk 1]
```diff
@@ -434,13 +434,13 @@ func runInit(i *initData, out io.Writer) error {
 		}
 
 		// PHASE 2: Generate kubeconfig files for the admin and the kubelet
-		glog.V(2).Infof("[init] generating kubeconfig files")
+		glog.V(2).Infof("[init] generating KubeConfig files")
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/init.go]: @yagonobre we should do the opposite:
KubeConfig -> kubeconfig.

the official naming is lowercase `kubeconfig` in the docs:

https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/
https://kubernetes.io/docs/search/?q=kubeconfig

**Review by neolit123 — COMMENTED** [review]
thanks @yagonobre 

please fix this typo:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/certs/certlist.go#L239

bellow are some more kubernetes -> Kubernetes actions.
these are found by grep (file : line number) / line numbers might not be exact:

```

constants/constants.go:382:	return nil, fmt.Errorf("Unsupported or unknown kubernetes version(%v)", kubernetesVersion)
constants/constants_test.go:154:			expectedError:     fmt.Errorf("Unsupported or unknown kubernetes version(1.99.0)"),
cmd/reset.go:143:	glog.V(1).Info("[reset] removing kubernetes-managed containers")

phases/upgrade/compute.go:305:		fmt.Printf("[upgrade/versions] WARNING: No recommended etcd for requested kubernetes version (%s)\n", kubernetesVersion)

preflight/checks.go:560:// KubernetesVersionCheck validates kubernetes and kubeadm versions
preflight/checks.go:571:// Check validates kubernetes and kubeadm versions
preflight/checks.go:573:	glog.V(1).Infoln("validating kubernetes and kubeadm version")
preflight/checks.go:586:		return nil, []error{fmt.Errorf("couldn't parse kubernetes version %q: %v", kubever.KubernetesVersion, err)}
preflight/checks.go:595:		return []error{fmt.Errorf("kubernetes version is greater than kubeadm version. Please consider to upgrade kubeadm. kubernetes version: %s. Kubeadm version: %d.%d.x", k8sVersion, kadmVersion.Components()[0], kadmVersion.Components()[1])}, nil
preflight/checks.go:626:			return nil, []error{fmt.Errorf("couldn't parse kubernetes version %q: %v", kubever.KubernetesVersion, err)}
preflight/checks_test.go:676:			t.Errorf("KubeletVersionCheck: unexpected warnings for kubelet version %q and kubernetes version %q. Warnings: %v", tc.kubeletVersion, tc.k8sVersion, warnings)
preflight/checks_test.go:678:			t.Errorf("KubeletVersionCheck: expected warnings for kubelet version %q and kubernetes version %q but got nothing", tc.kubeletVersion, tc.k8sVersion)
preflight/checks_test.go:680:			t.Errorf("KubeletVersionCheck: unexpected errors for kubelet version %q and kubernetes version %q. errors: %v", tc.kubeletVersion, tc.k8sVersion, errors)
preflight/checks_test.go:682:			t.Errorf("KubeletVersionCheck: expected errors for kubelet version %q and kubernetes version %q but got nothing", tc.kubeletVersion, tc.k8sVersion)
util/config/common.go:131:		return fmt.Errorf("couldn't parse kubernetes version %q: %v", cfg.KubernetesVersion, err)

features/features.go:90:					"the requested kubernetes version (%s) is incompatible with the %s feature gate, which needs %s as a minimum",

```

**Comment by neolit123** [comment]
@yagonobre 
thanks for the update.
please squash the commits. also, you need to run ./hack/update-gofmt.sh to update the formatted files (requires go 1.11+)

**Comment by yagonobre** [comment]
@neolit123 done

**Comment by neolit123** [comment]
thanks
/lgtm
/assign @fabriziopandini

**Comment by yagonobre** [comment]
@neolit123 rebased

**Comment by neolit123** [comment]
@yagonobre 
needs rebase again, but LGTM.

**Comment by neolit123** [comment]
@yagonobre i'm not sure what the CI errors is.
try running ./hack/update-bazel.sh locally.

**Comment by yagonobre** [comment]
@neolit123 yes, the rebase added a new dependency and I forgot the update bazel. I'll update soon.

**Review by timothysc — APPROVED** [review]
/approve

**Comment by neolit123** [comment]
@yagonobre sorry this needs quick rebase and then we can quickly LGTM.

**Comment by yagonobre** [comment]
@neolit123 done

**Pull request merged**